src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/SwaggerUI.cs

```diff
@@ -73,10 +77,16 @@ public ISwaggerUI AddServer(HttpRequest req, string routePrefix, IOpenApiConfigu
         }
 
         /// <inheritdoc />
-        public async Task<ISwaggerUI> BuildAsync()
+        public async Task<ISwaggerUI> BuildAsync(IUiCustomizationProvider uiCustomizationProvider = null)
         {
             var assembly = Assembly.GetExecutingAssembly();
 
+            if (uiCustomizationProvider != null)
```

Comment:
Refer to: #48

Reply:
This is a public method and it is possible someone could have overridden the default OpenApiHttpTriggerContext with their own and is passing in null (or using the parameterless method since uiCustomizationProvider is optional)

Reply:
Oh, you're right.